telegram-purple.c

```diff
@@ -532,23 +530,33 @@ static gulong chat_conversation_typing_signal = 0;
 static void tgprpl_login (PurpleAccount * acct) {
   info ("tgprpl_login(): Purple is telling the prpl to connect the account");
   
+  struct tgl_state *TLS = tgl_state_alloc ();
   PurpleConnection *gc = purple_account_get_connection (acct);
-  
   gc->flags |= PURPLE_CONNECTION_NO_BGCOLOR;
-  
-  struct tgl_state *TLS = tgl_state_alloc ();
   connection_data *conn = connection_data_init (TLS, gc, acct);
   purple_connection_set_protocol_data (gc, conn);
+  tgl_set_ev_base (TLS, conn);
 
   TLS->base_path = get_config_dir (purple_account_get_username (acct));
-  tgl_set_download_directory (TLS, get_download_dir(TLS));
   debug ("base configuration path: '%s'", TLS->base_path);
-  
+  g_mkdir_with_parents (TLS->base_path, 0700);
+
+  const gchar *base = !g_strcmp0 (purple_core_get_ui(), "BitlBee")
```

Comment:
Is this saying that, if the UI is bitlbee, that it should ignore the config'd path that's set in the compile options?